tests/rest/client/test_relations.py

```diff
@@ -278,16 +278,16 @@ def test_repeated_paginate_relations(self):
         found_event_ids.reverse()
         self.assertEquals(found_event_ids, expected_event_ids)
 
-    def test_pagination_from_sync_messages(self):
-        """Pagination tokens from sync and messages should work."""
+    def test_pagination_from_sync_and_messages(self):
+        """Pagination tokens from /sync and /messages can be used to paginate /relations."""
         channel = self._send_relation(RelationTypes.ANNOTATION, "m.reaction", "A")
         self.assertEquals(200, channel.code, channel.json_body)
         annotation_id = channel.json_body["event_id"]
         # Send an event after the relation events.
         self.helper.send(self.room, body="Latest event", tok=self.user_token)
 
-        # Request sync, but limit the timeline to get a pagination token before
-        # the reaction.
+        # Request /sync, limiting it such that only the latest event is returned
+        # (and not the relation).
```

Comment:
Aha, that's better. I would (rightly or wrongly!) have thought this would have been 'after' the reaction, not 'before' as it said before. This clears that up.